src/convertToObject.js

```diff
@@ -7,6 +7,20 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `array`, `object` `arr`, `obj`, `result`. It will decrease code readability. Consider using a more descriptive name that reflects the content or purpose of the variable, such as `styleObject`.